hdr/sqlite_modern_cpp.h

```diff
@@ -848,6 +846,29 @@ namespace sqlite {
 	 inline void store_result_in_db(sqlite3_context* db, const std::u16string& val) {
 		 sqlite3_result_text16(db, val.data(), -1, SQLITE_TRANSIENT);
 	}
+
+	// Other integer types
+	 template<class Integral, class = typename std::enable_if<std::is_integral<Integral>::value>::type>
+	 inline database_binder& operator <<(database_binder& db, const Integral& val) {
+		return db << static_cast<sqlite3_int64>(val);
```

Comment:
This will store `int` values as 64 bit values.
Please restrict the template with a condition like this. `is_integral<Integral> and sizeof(Integral) > sizeof(int)`.
And overload another template that stores `int` values and has condition like this `is_integral<Integral> and sizeof(integral) == sizeof(int)`